addon/components/au-main-header.js

```diff
@@ -4,7 +4,6 @@ import { action } from '@ember/object';
 export default class AuMainHeader extends Component {
   @action
   headerLinkFocus() {
-    // Focus content window
-    document.querySelectorAll('#main')[0].focus();
+    document.querySelector('#main')?.focus();
```

Comment:
Should we display a warning in development builds? I think this setup is here for accessibility reasons.